WordPress/src/main/res/layout/stats_widget_layout.xml

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/stats_widget_outer_container"
+    android:background="@drawable/stats_white_background"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/blog_title"
+        style="@style/StatsModuleTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:gravity="center_horizontal|center_vertical"
+        android:layout_margin="4dip"
+        android:text="@string/stats_widget_name" >
+    </TextView>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:orientation="horizontal"
+        android:background="@color/grey_lighten_30" />
+
+    <LinearLayout
+        android:id="@+id/stats_widget_values_container"
+        android:layout_height="50dp"
+        android:layout_width="match_parent"
+        android:orientation="horizontal">
+
+        <!-- Views -->
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="1"
+            android:orientation="vertical"
+            android:padding="@dimen/margin_medium"
+            android:gravity="center"
+            android:background="@drawable/stats_visitors_and_views_button_white"
+            android:clickable="false">
+
+            <LinearLayout
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                android:gravity="center"
+                android:orientation="horizontal">
+
+                <ImageView
+                    android:layout_width="12dp"
```

Comment:
There are a lot of instances of hard-coded `12dp` and `3dp` in this layout. Can they be made into `dimens` values instead?